deepchem/models/tests/test_dftxc.py

```diff
@@ -0,0 +1,46 @@
+import deepchem as dc
+from deepchem.models.dft.dftxc import XCModel
+from deepchem.data.data_loader import DFTYamlLoader
+import pytest
+import numpy as np
+import tempfile
+
+
+@pytest.mark.dqc
+def test_dftxc_eval():
+    inputs = 'deepchem/models/tests/assets/test_dftxcdata.yaml'
+    data = DFTYamlLoader()
+    model_dir = tempfile.mkdtemp()
+    dataset = (data.create_dataset(inputs))
+    model = XCModel("lda_x",
+                    batch_size=1,
+                    log_frequency=1,
+                    mode="classification",
+                    n_tasks=2,
+                    model_dir=model_dir)
+    loss = model.fit(dataset, nb_epoch=1, checkpoint_interval=1)
+    assert loss < 0.001
+    reload_model = XCModel("lda_x",
+                           batch_size=1,
+                           log_frequency=1,
+                           mode="classification",
+                           n_tasks=2,
+                           model_dir=model_dir)
+    reload_model.restore()
+    predict = model.predict(dataset)
+    r_predict = reload_model.predict(dataset)
+    assert np.all(predict == r_predict)
+    metric = dc.metrics.Metric(dc.metrics.mae_score)
+    scores = model.evaluate(dataset, [metric])
+    assert scores['mae_score'] < 0.3
+
```

Comment:
Documenting the offline discussion, we need to add unit tests for the case where `batch_size != 1` and another unit test for testing prediction on data where the true value is not known

Reply:
done